reference/platform-app-yaml.md

```diff
@@ -390,8 +398,10 @@ relationships:
     solr: "solr:solr"
     redis: "redis:redis"
 web:
-    document_root: "/"
-    passthru: "/index.php"
+    locations:
+        "/":
+            root: "public"
+            passthru: "/index.php"
```

Comment:
There are two relative paths here, one starts with `/`, the other doesn't. Would it work to make them consistent?